scripts/import_data.py

```diff
@@ -0,0 +1,480 @@
+#!/usr/bin/env python3
+
+import argparse
+import ast
+import datetime
+import ipaddress
+import json
+import random
+import re
+import subprocess
+import sys
+import uuid
+
+try:
+    import mysql.connector
+    from mysql.connector.connection import MySQLCursor
+    import pandas as pd
+    import yaml
+    import xlrd  # used by pandas
```

Comment:
Why is this import required?
We don't use xlrd here

Reply:
xlrd is used by pandas read_excel() method. I think it's clearer if we inform the user that it's missing at the start of the script, instead of later, when things are already running.
Improved the comment to make it clearer